daemon/algod/api/server/v2/account_test.go

```diff
@@ -102,7 +102,7 @@ func TestAccount(t *testing.T) {
 	b := a.WithUpdatedRewards(proto, 100)
 
 	addr := basics.Address{}.String()
-	conv, err := AccountDataToAccount(addr, &b, map[basics.AssetIndex]string{}, round, a.MicroAlgos)
+	conv, err := AccountDataToAccount(addr, &b, map[basics.AssetIndex]string{}, round, &proto, a.MicroAlgos)
```

Comment:
@tsachiherman : `proto` is **ConsensusFuture**. That seems like a reasonable option.